Developer-Guide.md

```diff
@@ -27,23 +29,32 @@
 
 # Warning
 
-This document is written **specifically for developers**.
+This document is written **specifically for developers**: it is not intended for end users.
 
 # Assumptions
 
 - You are working on a non-critical test or development system.
-- You already have the following installed:
-  - [Docker](https://www.docker.com/).
-  - [golang](https://golang.org/dl) version 1.8.3 or newer.
-  - `make`.
-  - `gcc` (required for building the shim and runtime).
-
-- You have installed the Clear Containers `linux-container` and `qemu-lite`
-  packages containing the guest kernel images and hypervisor. These packages
-  are automatically installed when you install Clear Containers, but can be
-  installed separately as well:
-
-    https://github.com/clearcontainers/runtime/wiki/Installation
+
+# Initial setup
+
+The recommended way to create a development environment is to first install the
+packaged versions of the Kata Containers components to create a working
+system:
+
+  * [Fedora*](install/fedora-installation-guide.md)
+  * [Ubuntu*](install/ubuntu-installation-guide.md)
+
+The installation guide instructions will install all required Kata Containers
+components, plus Docker*, the hypervisor, and the Kata Containers image and
```

Comment:
In general, I think all of our documentation assumes that all someone will ever want to do is setup Kata with Docker.  Its a good step, but I think we should be a bit more general, or at least include better install directions for Kubernetes development (this is an aside -- nothing we should address in this PR...).

Reply:
Agreed. I think that's a post release concern ;)